app/src/main/res/layout/profile_progress_activity.xml

```diff
@@ -1,7 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/profile_progress_fragment_placeholder"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  tools:context=".profileprogress.ProfileProgressActivity" />
+  android:orientation="vertical">
+
+  <include
+    android:id="@+id/profile_progress_activity_toolbar"
+    layout="@layout/toolbar" />
+
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <FrameLayout
+      android:id="@+id/profile_progress_fragment_placeholder"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      tools:context=".profileprogress.ProfileProgressActivity" />
```

Comment:
This should be introduced in top parent layout

Reply:
Done.